src/main/java/mobi/appcent/helium/model/hotspot/Witness.java

```diff
@@ -0,0 +1,20 @@
+package mobi.appcent.helium.model.hotspot;
+
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+/**
+ * Created by erenalpaslan on 27.10.2022
+ */
+@Data
+@AllArgsConstructor
+public class Witness {
+    private String timestamp;
+    private int signal;
```

Comment:
primitives vs objects :) Do you prefer primitives for a reason? I am just asking to understand.